build_tools/packaging/linux/README.md

```diff
@@ -0,0 +1,28 @@
+Prerequisites:
+
+Python version required:  python 3.12.3
```

Comment:
This is a very specific Python version. I presume other versions would work too? Maybe relax this to Python 3.10+ or 3.11+?